JavaScriptFileType.js

```diff
@@ -39,17 +39,15 @@ var JavaScriptFileType = function(project) {
 
     this.pseudos = {};
 
-    if (typeof project.pseudoLocale === "string") {
-        project.pseudoLocale = [project.pseudoLocale];
-    }
-
     // generate all the pseudo bundles we'll need
-    project.pseudoLocale && project.pseudoLocale.forEach(function(locale) {
-        var pseudo = this.API.getPseudoBundle(locale, this, project);
-        if (pseudo) {
-            this.pseudos[locale] = pseudo;
+    if (project.pseudoLocales && typeof project.pseudoLocales == 'object'){
```

Comment:
Need to be able to deal with the situation where it is an array OR an object.

Reply:
looks like it never happens to return the `array` type. isn't it?
https://github.com/iLib-js/loctool/blob/nopseudo/lib/Project.js#L86
https://github.com/iLib-js/loctool/blob/nopseudo/lib/PseudoFactory.js#L227

Reply:
In the "nopseudo" branch, then yes.
But in an older version of loctool, then no

Reply:
I updated to support both cases and check if these are correctly generating pseudo translation data